framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -565,10 +565,13 @@ protected void warnUnneededSuppressions() {
     protected void printOrStoreMessage(
             Diagnostic.Kind kind, String message, Tree source, CompilationUnitTree root) {
         assert this.currentRoot == root;
+        StackTraceElement[] trace = new RuntimeException().getStackTrace();
```

Comment:
Use `Thread.getStackTrace()` https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html#getStackTrace() instead of creating that exception object.

Reply:
I have used Thread.currentThread().getStackTrace() to acquire the stack trace.